bulbs/recirc/mixins.py

```diff
@@ -43,11 +43,9 @@ def get_recirc_content(self, published=True, count=3):
         """
         query = self.get_query()
 
-        # check if query has included_ids & if there are any ids in it
-        try:
-            if len(query['included_ids']) == 0:
-                raise KeyError
-        except KeyError:
+        # check if query has included_ids & if there are any ids in it,
+        # in case the ids have been removed from the array
+        if not query.get('included_ids') or len(query.get('included_ids')) == 0:
```

Comment:
this also exposed a bug where the master object would show up in the result sometimes